dev/VSIX/ProjectTemplates/Neutral/CppWinRT/RuntimeComponent/pch.h

```diff
@@ -1,16 +1,26 @@
-// Copyright (c) Microsoft Corporation and Contributors.
-// Licensed under the MIT License.
-
 #pragma once
+#include <windows.h>
 #include <unknwn.h>
+#include <restrictederrorinfo.h>
+#include <hstring.h>
+
+// Undefine GetCurrentTime macro to prevent
+// conflict with Storyboard::GetCurrentTime
+#undef GetCurrentTime
+
 #include <winrt/Windows.Foundation.h>
 #include <winrt/Windows.Foundation.Collections.h>
+#include <winrt/Windows.ApplicationModel.Activation.h>
+#include <winrt/Windows.UI.Xaml.Interop.h>
 #include <winrt/Microsoft.UI.Composition.h>
 #include <winrt/Microsoft.UI.Xaml.h>
 #include <winrt/Microsoft.UI.Xaml.Controls.h>
 #include <winrt/Microsoft.UI.Xaml.Controls.Primitives.h>
 #include <winrt/Microsoft.UI.Xaml.Data.h>
+#include <winrt/Microsoft.UI.Xaml.Interop.h>
 #include <winrt/Microsoft.UI.Xaml.Markup.h>
+#include <winrt/Microsoft.UI.Xaml.Media.h>
 #include <winrt/Microsoft.UI.Xaml.Navigation.h>
+#include <winrt/Microsoft.UI.Xaml.Shapes.h>
 #include <winrt/Microsoft.UI.Dispatching.h>
 #include <wil/cppwinrt_helpers.h>
```

Comment:
>cppwinrt_helpers [](http://example.com/codeflow?start=14&length=16)

Does this include `<wil/cppwinrt.h>` to enable WIL/cppwinrt interop?

Reply:
No. That header file's sole purpose is to support async calls (WIL is where `resume_foreground` ended up landing). More generally, I recall there was resistance in the past to the project templates automatically opting in developers to using WIL which is why we limit our usage to that one header file. If the functionality hadn't ended up in WIL then we wouldn't even have a Nuget package reference to it in the first place.